### PR TITLE
Fix null types

### PR DESCRIPTION
**Pull request by xwatkins**

### CODE REVIEW

File: src/model/FreeText.tsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ export const FreeText: React.FC<FreeTextDataProps> = ({
   type,
   includeTitle = false,
 }) => {
-  if (!data.comments) {
+  if (!data.comments || !data.comments.find(d => d.commentType === type)) {
```
Comment (dlrice) [inline src/model/FreeText.tsx]: `some` might fit a better here.

File: src/model/CatalyticActivity.tsx [hunk 2]
```diff
@@ -24,11 +24,14 @@ type CatalyticActivityProps = {
 export const CatalyticActivity: React.FC<CatalyticActivityProps> = ({
   data,
 }) => {
-  if (!data.comments) {
+  if (
+    !data.comments ||
+    !data.comments.find(d => d.commentType === FreeTextType.CATALYTIC_ACTIVITY)
```
Comment (dlrice) [inline src/model/CatalyticActivity.tsx]: `some` might fit a better here.

**Comment by dlrice** [comment]
Could be worth adding some model data (`src/model/__tests__/modelData.json`) that has some missing information to make sure it's failing gracefully / rendering nothing (ie no unnecessary commas).

**Comment by xwatkins** [comment]
Using `some`, made a Jira for test cases

**Pull request merged**